changelog/1.38.0.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "1.38.0"
+description: "Released on 01/04/2023"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.38.0.
+
+### Features ✨
+
+- Individual image tags can now be decommissioned. Existing workspaces using a
+  decommissioned tag will be migrated to use the default tag upon rebuild. Adds
+  new organization-level roles: Organization Super Manager, Organization Image
+  Manager, and Organization Importer. These roles allow finer-grained access
+  control around image and registry management. For more information, see
+  Organization Roles.
+  > ℹ️ ️Migration: existing users with the Organization Manager role will be
+  > migrated to Organization Super Manager, and users with the Organization
+  > Member role will be migrated to Importer. Both of these changes will result
+  > in no effective permission changes for existing users.
+- Improves web terminal reconnection by leveraging GNU Screen if available
```

Comment:
In case we want to add the link:

```suggestion
- Improves web terminal reconnection by leveraging [GNU Screen](https://www.gnu.org/software/screen/) if available
```

It looks like it would also be an option to use `[GNU Screen]` and then put the link further down if we want to avoid messing with the raw text flow (stolen from https://raw.githubusercontent.com/coder/docs/edfbbea5140db13d2d210e67374b0ad537be418e/changelog/1.27.0.md):

```
<!-- markdownlint-disable MD044 -->
[GNU Screen]:
  https://www.gnu.org/software/screen/
<!-- markdownlint-enable MD044 -->
```